src/index.html

```diff
@@ -12,6 +12,149 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form 
+      class="form1" 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="title">
+          <label for="s">Surname: 
+            <input 
+              name="sur" 
+              type="text" 
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="title">
+          <label for="n">Name: 
+            <input 
+              name="name" 
+              type="text" 
+              required  
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="title">
+          <label for="l">How old are You? 
+            <input 
+              name="num" 
+              type="number" 
+              value="12" 
+              min="0" 
+              max="100"
+            >
+          </label>
+        </div>
+        <div class="title"> 
+          <label for="l0">Full date of birth: 
+            <input 
+              name="date" 
+              type="date" 
+              placeholder="mm/dd/yyyy"
+            >
+          </label>
+        </div>
+        <label for="l1">I accept the term of the agreement 
+          <input 
+            name="check" 
+            type="checkbox"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="title">
+          <label for="l2">E-mail: 
+            <input 
+              name="email" 
+              type="email" 
+              placeholder="email@example.com" 
+              required
+            >
+          </label>
+        </div>
+        <label for="l3">Password: 
+          <input 
+            name="pass" 
+            type="password" 
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you! </legend>
+        <div class="title">
+          <label for="l4">Do you love cats? 
+            <input 
+              type="radio" 
```

Comment:
Use meaningful names for elements.

[More info.](https://en.bem.info/methodology/naming-convention/)